README.md

```diff
@@ -1 +1,188 @@
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- User login form
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link (not yet deployed)](#)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Installation of Git
+installation of NPM
+a working browser
+live server
+Internet connection 
+A code editor like Visual Studio Code are required.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+```sh
+  cd my-folder(your working machine)
+  git clone git@github.com:Fentahunn/finish_mobile_version.git
+
+```
+
+
+### Install
+
+Install this project with:
+
+
+```sh
+  cd my-project
+  To install Webhint
+  npm init -y
+  npm install --save-dev hint@7.x
+  To install Stylelint
+  npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x
+
+```
+
+### Usage
+
+To run the project, open the project by launching the Live Server plugging in visual studio code.
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Fentahun Agmas**
+
+- GitHub: [@githubFentahun](https://github.com/Fentahunn)
+- Twitter: [@twitterFentahun](https://twitter.com/FentahunAgmas)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Responsive website for different screen sizes like desktop or tablet .**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/Fentahunn/finish_mobile_version/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I am greateful for microvese community!
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+
+- **[What is Mobile-first?]**
+
+  - [The objective is to prioritize the mobile user experience by ensuring that CSS is designed to cater to smaller devices without the need to sift through styles intended for desktop devices.]  
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.

```

Comment:
- [ ] Please 🙏🏻  add the `MIT` license properly because including the **MIT** *License* in your `GitHub` repository **README** is crucial for several reasons. Firstly, the **MIT** License is a permissive `open-source` license that grants users the freedom to use, modify, and distribute the software without legal restrictions. By choosing the *MIT* License 🗒️, you are encouraging collaboration, inviting other developers to contribute, and fostering a vibrant open-source community. Additionally, using the *MIT* License helps protect your work by clearly defining the terms and conditions under which it can be used, providing a level of legal 🚨 clarity and ensuring that proper attribution is given. Furthermore, the widespread popularity and familiarity of the *MIT* License make it easy for others to understand and comply with the licensing requirements, which ultimately promotes the adoption and usage of your software 💻 .